config/config_test.go

```diff
@@ -79,6 +87,18 @@ func (s *ConfigTestSuite) TestGracePeriodSeconds() {
 	s.Equal(&g, GracePeriodSeconds())
 }
 
+func (s *ConfigTestSuite) TestBlacklistedNamespacesEnv() {
+	blns := []string{"namespace3", "namespace4"}
+	envname := "KUBEMONKEY_BLACKLISTED_NAMESPACES"
+	defer os.Setenv(envname, os.Getenv(envname))
+	os.Setenv(envname, "namespace3 namespace4")
```

Comment:
Could we do:

```
os.Setenv(envname, strings.Join(blns, " ")
```

This ensures `blns` is the source single source of truth

Reply:
good spot, sorted